etc/picongpu/hydra-hzdr/default_picongpu.profile.example

```diff
@@ -0,0 +1,65 @@
+# Modules #####################################################################
+#
+
+if [ -f /etc/profile.modules ]
+then
+        . /etc/profile.modules
+        module purge
+#       export MODULES_NO_OUTPUT=1
+
+        # Core Dependencies
+        module load gcc/5.3.0
+        module load cmake/3.7.2
+        module load boost/1.62.0
+        module load openmpi/1.8.6
+        module load numactl
+
+        # Plugins (optional)
+        module load pngwriter/0.6.0
+        module load hdf5-parallel/1.8.15 libsplash/1.6.0
+
+        # either use libSplash or ADIOS for file I/O
+        #module load adios/1.10.0
+
+        # Debug Tools
+        #module load gdb
+        #module load valgrind/3.8.1
+
+#       unset MODULES_NO_OUTPUT
+fi
+
+# Architecture flags ##########################################################
+#
+export CXXFLAGS="$CXXFLAGS -march=ivybridge -mtune=ivybridge"
```

Comment:
Future: My general idea is more to add a configuration env option such as `PIC_ARCH` (for `pic-configure`) for queues_clusters in this file.